app/controllers/symbols_controller.rb

```diff
@@ -18,7 +18,13 @@ def index
     if ontology && (ontology.is?('OWL2') || ontology.is?('OWL'))
       begin
         symbols = ontology.symbol_groups
+        request = <<-SQL
+          id NOT IN (SELECT DISTINCT(parent_id) FROM e_edges
+           UNION SELECT DISTINCT(child_id) FROM e_edges)
+        SQL
+        extra_notes = ontology.symbol_groups.where(request)
```

Comment:
I suppose you meant `extra_nodes` with a "d" ;)
In what way are these "extra"? It seems like they are in a subclass hierarchy somewhere between the child and the parent, but I'm not sure if my guess is correct.